templates/CRM/CiviAwards/Form/AwardReview.tpl

```diff
@@ -90,13 +90,13 @@
       {* End if error message *}
 
       {* Form fields section Starts *}
-      {foreach from=$elementNames item=elementName}
+      {foreach from=$elementData item=element}
         <div class="form-group {$form_group_class}">
-          <label class="{$form_group_label_class}">{$form.$elementName.label}</label>
+          <label class="{$form_group_label_class}">{$form[$element.name].label}</label>
           {if $isReviewFromSsp}
-            <div class="ssp-form-control-description text-muted"> Lorem ipsum dolor sit. Sit dolor ipsum lorem porem. </div>
+            <div class="ssp-form-control-description text-muted"> {$element.help_post} </div>
```

Comment:
TBH. I feel that this should be `<div class="ssp-form-control-description text-muted"> {$element.help_pre} </div>`

Reply:
`Pre Help` are supposed to be help fields displayed before a form field in Civicrm. Post help are displayed in tooltips when the icon in front of the field is clicked. I think the choice is ours here since this is a bit custom. I've exposed both fields and we can use both as we deem fit.